src/user-management/interfaces/authenticate-with-code-and-verifier-options.interface.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  AuthenticateWithOptionsBase,
+  SerializedAuthenticateWithPKCEBase,
+} from './authenticate-with-options-base.interface';
+
+export interface AuthenticateWithCodeAndVerifierOptions
+  extends AuthenticateWithOptionsBase {
+  codeVerifier: string;
+  code: string;
+  invitationToken?: string;
+}
+
+export interface AuthenticateUserWithPKCECredentials {
```

Comment:
is this interface doing anything for us? (i noticed that authenticateWithCode doesn't need an interface for the client_id)